### DISCUSSION TITLE
Compatibility Inquiry

### DISCUSSION THREAD

**Discussion by julio-ccatb** [jaredhendrickson13/pfsense-api#413] [Q&A] [answered]
**Project Compatibility Inquiry: Is pfsense-api compatible with pfSense 2.6.0-RELEASE and previous versions?**

Hello,

I am currently using pfSense version 2.6.0-RELEASE for my network infrastructure. I am interested in using the pfsense-api for a specific functionality, but I need to know if it is compatible with pfSense 2.6.0-RELEASE or any previous version.

Could you please confirm if pfsense-api is known to work without issues on pfSense 2.6.0-RELEASE and its previous versions? If it does, are there any specific considerations or configuration settings I should be aware of to ensure a smooth integration?

Thank you in advance for your help.

**Comment by jaredhendrickson13** [comment, marked as answer]
The last version to support pfSense 2.6.0-RELEASE was [v1.5.4](https://github.com/jaredhendrickson13/pfsense-api/releases/tag/v1.5.4). v1.5.x is still supported for now, but only receives security updates for known CVEs. It can be installed with the following command:

```
pkg add https://github.com/jaredhendrickson13/pfsense-api/releases/download/v1.5.4/pfSense-2.6-pkg-API.txz && /etc/rc.restart_webgui
```

pfSense 2.6.0-RELEASE is not compatible with v1.6.x and later as it contains changes required to be compatible with PHP 8 and the new configuration access functions introduced in pfSense 2.7.0-RELEASE.
